src/js/pensions/config/form.js

```diff
@@ -26,6 +27,12 @@ const formConfig = {
     }
   },
   chapters: {
+    applicantInformation: {
```

Comment:
We should write some tests for this, too. I think there are existing tests for this page, we should follow that and adapt them to fields being shown here.